src/main.cc

```diff
@@ -498,7 +497,10 @@ int main( int argc, char ** argv )
         return -1;
 
       QString configFile = Config::getConfigFileName();
-      renameAtomically( configFile, configFile + ".bad" );
+      QFile::rename( configFile,
+                     configFile % QStringLiteral( "." )
+                       % QDateTime::currentDateTime().toString( QStringLiteral( "yyyyMMdd_HHmmss" ) )
```

Comment:
Time stamped bad/backup files, instead of overwriting the same one.

Reply:
Can QSaveFile be used to save the backupfile to `configFile+".bad"`

Reply:
no